AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/max/nativead/ManualNativeAdActivity.kt

```diff
@@ -38,6 +40,7 @@ class ManualNativeAdActivity : BaseAdActivity() {
             .setIconImageViewId(R.id.icon_image_view)
             .setMediaContentViewGroupId(R.id.media_view_container)
             .setOptionsContentViewGroupId(R.id.options_view)
+                .setStarRatingContentViewGroupId( R.id.star_rating_view)
```

Comment:
nit: Make sure function calls are aligned
```suggestion
            .setStarRatingContentViewGroupId( R.id.star_rating_view)
```

Reply:
Please make it a habit to run auto-formatter (shift + cmd + option + L) when developing

Reply:
Ok, just ran that and actually caught some other things.